<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryLanewayForm</class>
 <widget class="QWidget" name="queryLanewayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>89</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>85</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查询巷道</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>393</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>巷道名称:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="queryBtn">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="queryLanewayCmb">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>queryBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
